libs/rand/random.go

```diff
@@ -141,9 +139,7 @@ func RandPerm(n int) []int {
 // Rand methods
 
 func (r *Rand) Seed(seed int64) {
```

Comment:
we'll need to make sure we seed each instance of Rand throughout the code

Reply:
this is done for seeding:
```
for i := 0; i < 8; i++ {                                                     
         seed |= uint64(bz[i])                                                    
         seed <<= 8                                                               
}
```
you want to replace this?

Reply:
we only need OS random in one / two places, in other we can just seed with the timestamp I believe

Reply:
okay. Let me see if I can help with that.